pragma.dd

```diff
@@ -57,11 +57,19 @@ $(H3 $(LEGACY_LNAME2 Predefined-Pragmas, predefined-pragmas, Predefined Pragmas)
 
         $(P All implementations must support these, even if by just ignoring
         them:
+
+        $(UL
+            $(LI $(LINK2 #inline, pragma inline))
```

Comment:
Need `$(HASH)` instead of `#`. Run `make pdf` to make sure the pdf makes it through.